packages/grid/README.md

```diff
@@ -32,41 +29,44 @@
 </script>
 ```
 
-[<img src="https://raw.githubusercontent.com/vaadin/vaadin-grid/master/screenshot.png" alt="Screenshot of vaadin-grid, using the default Lumo theme">](https://vaadin.com/components/vaadin-grid)
+[<img src="https://raw.githubusercontent.com/vaadin/web-components/master/packages/grid/screenshot.png" alt="Screenshot of vaadin-grid">](https://vaadin.com/docs/latest/ds/components/grid)
 
 ## Installation
 
-Install `vaadin-grid`:
+Install the component:
 
 ```sh
-npm i @vaadin/grid --save
+npm i @vaadin/grid
 ```
 
-Once installed, import it in your application:
+Once installed, import the component in your application:
 
 ```js
-import '@vaadin/grid/vaadin-grid.js';
+import '@vaadin/grid';
```

Comment:
Wouldn't you need to import other things as well (column, sort column)?

Reply:
Added the column imports.